tests/tests_pytorch/plugins/test_amp_plugins.py

```diff
@@ -193,3 +194,17 @@ def test_cpu_amp_precision_context_manager(tmpdir):
     assert isinstance(context_manager, torch.autocast)
     # check with str due to a bug upstream: https://github.com/pytorch/pytorch/issues/65786
     assert str(context_manager.fast_dtype) == str(torch.bfloat16)
+
+
+def test_amp_precision_plugin_parameter_validation():
+    MixedPrecisionPlugin("16-mixed", "cpu")  # should not raise exception
+    MixedPrecisionPlugin("bf16-mixed", "cpu")
+
+    with pytest.raises(MisconfigurationException):
```

Comment:
Can you add the `match=` argument to this?

Reply:
Done